Casks/graalvm-jdk17.rb

```diff
@@ -1,11 +1,11 @@
 cask "graalvm-jdk17" do
   arch arm: "aarch64", intel: "x64"
 
-  version "17.0.12"
-  sha256 arm:   "4cdfdc6c9395f6773efcd191b6605f1b7c8e1b78ab900ab5cff34720a3feffc5",
-         intel: "3ecac1471f3fa95a56c5b75c65db9e60ac4551f56eda09eb9da95e6049ea77d7"
+  version "17.0.13"
```

Comment:
We do not publish new builds where is this coming from?

Reply:
new builds require a clickthough on OTN

Reply:
I will make the change now, Thank you